tests/src/FunctionalJavascript/MessageFormTest.php

```diff
@@ -170,6 +179,9 @@ public function testDefaultFormNotChanged(): void {
     // Assert fields in autoreply.
     $this->assertTrue(strpos($captured_emails[1]['body'], 'Test subject') === FALSE);
     $this->assertTrue(strpos($captured_emails[1]['body'], 'Test message') === FALSE);
+
+    // Assert redirection was not changed.
```

Comment:
Again, misleading comment. Why is the redirect to the user/` page?

`Assert that non-corporate forms are being redirected to the homepage (which in the case of the test setup is the user page)`.

Reply:
to answer why? its drupal behaviour to redirect to that page if the user is logged and the destination is <front>, its not just in tests btw

so I would change "in the case of the test setup"

Reply:
Yes, but the test setup makes the homepage to be the user page cause one is not set up. So if somebody (me) is reading the test, doesn't know why that is and doesnt understand. So instead of spending 30 seconds reading and understanding the test, spends way way more.

Reply:
ok, fixed